### PR TITLE
createWithMixins deprecation doc

### PR DESCRIPTION
**Pull request by jmejias**

### CODE REVIEW

File: source/deprecations/v1.x.html.md [hunk 1]
```diff
@@ -1022,4 +1022,23 @@ c = Contact.create({ firstName: "John", lastName: "Doe" });
...
 Object.freeze(c);
 c.swapNames();  // EXCEPTION
-```
\ No newline at end of file
+```
+
+
+#### CreateWithMixins
+
+`Ember.createWithMixins` method has been deprecated in favor of create or extend.
```
Comment (rwjblue) [inline source/deprecations/v1.x.html.md]: Ember.Object.createWithMixins (not Ember.createWithMixins).

The replacement is to call Ember.Object.extend().create() (so extend AND create).

**Comment by jmejias** [comment]
@rwjblue fixed based on your comments, please revise thanks

**Comment by rwjblue** [comment]
Looks like merging your other PR made this one have a conflict. Would you mind rebasing?

**Comment by mixonic** [comment]
@jmejias this still seems to need a little effort for completion:

* Needs rebasing and for the commits to be squashed.
* Commit and issue should be prefixed with `[DOC release]` for robert can get it into the correct branches.

File: source/deprecations/v1.x.html.md [hunk 2]
```diff
@@ -1069,3 +1069,21 @@ c.swapNames();  // returns c
...
+
+});
+```
+
+Replace with code above with:
+
+```javascript
+var obj = Ember.Object.create({
+
+});
```
Comment (mixonic) [inline source/deprecations/v1.x.html.md]: I'd actually suggest:

```
var obj = Ember.Object.extend({
}).create();
```

As that is actually exactly what `createWithMixins` did.

Comment (jmejias) [inline source/deprecations/v1.x.html.md]: @mixonic thanks I will update

**Comment by jmejias** [comment]
@mixonic I rebased and cleaned up

**Comment by mixonic** [comment]
@jmejias please make it just one commit, with the prefix `[DOC release]`. For example this would be a great commit message:

```
[DOC release] Add
```

**Comment by mixonic** [comment]
This is great @jmejias. Sorry for misleading you with the `[DOC release]` stuff, this is the website not the ember codebase!

**Pull request merged**